stock_picking_comment_template_ext/README.rst

```diff
@@ -14,16 +14,16 @@ Picking Comments Ext
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Frpl--custom-lightgray.png?logo=github
-    :target: https://github.com/qrtl/rpl-custom/tree/12.0/stock_picking_comment_template_ext
+    :target: https://github.com/qrtl/rpl-custom/tree/13.0/stock_picking_comment_template_ext
```

Comment:
Revert the version to 12.0.

Reply:
OK, I do. (The psc's settings were left unchanged.)